### DISCUSSION TITLE
Cannot setting up copilot in WebStorm

### DISCUSSION THREAD

**Discussion by EpicHigh** [community/community#19538] [Copilot Conversations] [answered]
I currently use WebStorm version `2022.1.3` and Copilot version `1.1.27.1708`. I already subscribed to Copilot using my credit card.

#### 1. Click a tiny GitHub icon below to open this dialog

<img width="431" alt="Screen Shot 2565-06-27 at 15 41 33" src="https://user-images.githubusercontent.com/19851878/175898642-a8467ca4-1ae6-45d5-9874-6bdebb909321.png">

#### 2. Click `Copy and Open`, paste the code from a previous dialog and click `continue`

<img width="569" alt="Screen Shot 2565-06-27 at 15 41 49" src="https://user-images.githubusercontent.com/19851878/175898866-21142c4a-39c6-47cd-b25e-ccb1bcc1ed67.png">

#### 3. Click `Authorize GitHib Copilot Plugin`

<img width="695" alt="Screen Shot 2565-06-27 at 15 41 57" src="https://user-images.githubusercontent.com/19851878/175899166-607bc405-bd96-46cf-aee9-a68247f115f7.png">

#### 4. It shows that everything is success.

<img width="631" alt="Screen Shot 2565-06-27 at 15 42 02" src="https://user-images.githubusercontent.com/19851878/175899499-9bf52d04-c64a-440a-b753-96c48d1682b0.png">

#### 5. Return to my IDE, It shows the following message.

<img width="449" alt="Screen Shot 2565-06-27 at 15 42 09" src="https://user-images.githubusercontent.com/19851878/175899681-fb54fb2e-76c2-4c71-8125-50715a6da1ea.png">

#### 6. Click `Sign Up` and got 404 page

<img width="1440" alt="Screen Shot 2565-06-27 at 15 42 16" src="https://user-images.githubusercontent.com/19851878/175899939-78647669-fc5c-4703-93e9-871bcc21d2e3.png">

**Comment by davecheney** [comment]
@EpicHigh I'm sorry you're having problems. I suspect the issue is step 6, can you copy and paste the URL of the page that gave you the 404 (I suspect it has a errant trailing period)

**Reply by omrigee** [reply]
having the same issue, here it is:
https://github.com/features/copilot/signup

**Reply by davecheney** [reply]
@omrigee i'm sorry, that was an out of date notification, please go to 

https://github.com/github-copilot/signup

**Comment by davecheney** [comment, marked as answer]
@EpicHigh the correct url is

https://github.com/github-copilot/signup

**Comment by ChiPhanTlm** [comment]
Still not working in IntelliJ IDE, @davecheney I go to ur link but it only take me to the billing page while I'm already paid, the IDE still can't login

**Reply by davecheney** [reply]
@ChiPhanTlm please go to https://github.com/settings/copilot and complete the required steps.

**Reply by ChiPhanTlm** [reply]
@davecheney now it's working, thank you very much!!

**Reply by omrigee** [reply]
<img width="1095" alt="Screen Shot 2022-06-29 at 10 35 55" src="https://user-images.githubusercontent.com/44809100/176379112-fc6fb13a-f43d-4284-be72-bd174c602a1e.png">

already did, and yet I get the same "You dont have access to GitHub Copilot" popup

**Reply by davecheney** [reply]
@omrigee I can see that you tried to access copilot about 18 hours ago, and there was a problem with your billing (long story, copilot doesn't bill you anything, but the billing system tries to verify that it _could_, even though it won't -- this is dumb, we fixed it) which has since been resolved. I suggest trying again.

**Reply by omrigee** [reply]
thanks @davecheney, this issue is solved but now im facing another one:
I'm trying to login
![Screen Shot 2022-06-29 at 13 33 20](https://user-images.githubusercontent.com/44809100/176416659-865e5791-f80e-4b40-9178-40c57401a8ff.png)

and get 

![Screen Shot 2022-06-29 at 13 35 01](https://user-images.githubusercontent.com/44809100/176416706-eca8a662-bdc3-4510-a767-3db5b3b77096.png)

although im logged in 
![Screen Shot 2022-06-29 at 13 34 46](https://user-images.githubusercontent.com/44809100/176416903-e0a3d78c-fb41-4a04-89a7-e2c40b12de59.png)

**Reply by davecheney** [reply]
Can you pls try going to https://github.com/settings/applications, look for "Github for VS Code" and delete it, then logout/login. The application may be called "GitHub for Intellij", i'm not sure, I'm not an IJ user.

**Reply by omrigee** [reply]
@davecheney you are the real MVP! 
Thank you that works
